### PR TITLE
Add a new "Community" directory

### PR DESCRIPTION
**Pull request by jaeyounkim**
Add a new "Community" directory
Add description for a new "Community" directory

### CODE REVIEW

File: community/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+![Logo](https://storage.googleapis.com/model_garden_artifacts/TF_Model_Garden.png)
+
+# TensorFlow Community Models
+
+We provide a curated list of the GitHub repositories with machine learning models and implementations powered by TensorFlow 2.
+
+Unlike our "[official](https://github.com/tensorflow/models/tree/master/official)" and "[research](https://github.com/tensorflow/models/tree/master/research)" repositories, we do not maintain the GitHub repositories listed here. Their respective maintainers support them.
```
Comment (saberkun) [inline community/README.md:7]: Could we clarify what are "we"? TensorFlow Model Garden team or TensorFlow or Google teams?

File: community/README.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+## Models / Implementations
+
+| Use case | GitHub Repository | Paper | Description | Maintainer(s) |
+|----------|-------------------|-------|-------------|---------------|
+| Computer Vision | [Mask R-CNN](https://github.com/NVIDIA/DeepLearningExamples/tree/master/TensorFlow2/Segmentation/MaskRCNN) | Mask R-CNN (https://arxiv.org/abs/1703.06870) | A script and recipe to train the Mask R-CNN model | [NVIDIA](https://github.com/NVIDIA) |
```
Comment (saberkun) [inline community/README.md:13]: Why do we need to say "a script or recipe to train"? Can we just say Nvidia-optimized version of XXX?

Comment (saberkun) [reply community/README.md:13]: Should we ask Nvidia folks review?

File: community/README.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+| Computer Vision | [U-Net Medical Image Segmentation](https://github.com/NVIDIA/DeepLearningExamples/tree/master/TensorFlow2/Segmentation/UNet_Medical) | U-Net: Convolutional Networks for Biomedical Image Segmentation (https://arxiv.org/abs/1505.04597) | A script and recipe to train U-Net Medical | [NVIDIA](https://github.com/NVIDIA) |
+
+
+## Contributions
```
Comment (saberkun) [inline community/README.md:17]: Could we mention our review criteria and procedure?
What repos are expected here? I guess personal repos might not be expected at this moment.

File: community/README.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+If you have a suggestion for the community model repository, please submit a pull request.
```
Comment (saberkun) [inline community/README.md:18]: Who will be reviewers? How about adding our github ids?

File: community/README.md (hunk 2 above)
Comment (jaeyounkim) [reply community/README.md:13]: This description is from Nvidia's repository. I will add Nvidia as a reviewer.

File: community/README.md (hunk 3 above)
Comment (jaeyounkim) [reply community/README.md:17]: Contribution guidelines will be provided next week.

File: community/README.md (hunk 4 above)
Comment (jaeyounkim) [reply community/README.md:18]: The Model Garden team will be reviewers. There is no way for sending a direct/private message using a GitHub username. Other model zoos are also using a pull request to accept a new contribution.

File: community/README.md (hunk 1 above)
Comment (jaeyounkim) [reply community/README.md:7]: Good point. How about changing "We" to "This community repository"?

**Comment by jaeyounkim** [comment]
@nvpstr Please review this pull request since we want to add Nvidia's GitHub repositories to our curated list.

File: community/README.md (hunk 1 above)
Comment (saberkun) [reply community/README.md:7]: Maybe tensorflow/models owners

File: community/README.md (hunk 4 above)
Comment (saberkun) [reply community/README.md:18]: OK.

File: community/README.md (hunk 1 above)
Comment (alvarognvidia) [inline community/README.md:7]: To incentive users to try the community models, portrait the community models in a good light, and fully disclose Google’s involvement in maintaining these models, could we please adjust the language in line 7? If we add “Disclosure: Contributing companies are responsible for maintaining their repositories” we remove the negative connotation and clearly disclose Google’s involvement.

Comment (jaeyounkim) [reply community/README.md]: I replaced "We" with "This repository" since this expression is commonly used.

Comment (jaeyounkim) [reply community/README.md]: Thank you for your feedback. I've updated the table.

**Pull request merged**